api/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -165,7 +165,6 @@
    * @return this
    * @since 1.0.0
    */
-  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1772")
```

Comment:
Leave the annotation. That way it is clear it can be deleted, vs a deprecated method that was stable so we can't delete it.

Reply:
Done.